mne/gui/_coreg_gui.py

```diff
@@ -1352,16 +1382,26 @@ def _get_omitted_info(self):
         if self.model.hsp.n_omitted == 0:
             return "No points omitted"
         elif self.model.hsp.n_omitted == 1:
-            return "1 point omitted"
+            omit_str = "1 point"
         else:
-            return "%i points omitted" % self.model.hsp.n_omitted
+            omit_str = "%i points" % self.model.hsp.n_omitted
+
+        omit_str += " out of %d omitted (HSP: %d; EEG: %d)" % \
+                    (len(self.model.hsp.inst_points),
+                     self.model.hsp.n_omitted_types['HSP'],
+                     self.model.hsp.n_omitted_types['EEG'])
+        return omit_str
 
     def _omit_points_fired(self):
         distance = self.distance / 1000.
-        self.model.omit_hsp_points(distance)
+        self.model.omit_hsp_points(distance, reset=False)
 
     def _reset_omit_points_fired(self):
-        self.model.omit_hsp_points(0, True)
+        self.model.omit_hsp_points(0, reset=True)
+        self.model.apply_eeg_filter()  # Don't forget the EEG
+
+    def _use_eeg_locations_fired(self):
+        self.model.apply_eeg_filter()
```

Comment:
Couldn't `_use_eeg_locations_fired` be on the `CoregModel`? Then you would not have to manually trigger it in the tests?